<launch>
    <node pkg="nate_stretch_movement" type="explorer_node" name="Explorer_Node" output="screen">
        <rosparam file="$(find nate_stretch_movement)/params/right_wall_hugger_params.yaml" command="load"/>
    </node>
</launch>